### ISSUE TITLE
I want to use L0G-101086 with non-CM fractals

### ISSUE THREAD

**Issue by pickettd** [jacob-keller/L0G-101086#29]
**Is your feature request related to a problem? Please describe.**
The problem is that I want to use L0G-101086 with non-CM fractals

**Describe the solution you'd like**
I'd like L0G-101086 to support non-CM fractals the way it parses/uploads/announces raids and CM-fractals.

**Describe alternatives you've considered**
I hacked together support for some non-CM fractals in [my fork](https://github.com/pickettd/L0G-101086) by using the slightly-outdated boss ids from [this wiki page](https://wiki.guildwars2.com/wiki/User:InAGiiZ/npcids#PvE_Fractal_NPCs). It is working but it is an incomplete list and I know the hack does not match the architecture of this project.

**Additional context**
I'm pretty sure that the correct way to handle support for this is in updating the boss id tables in simpleArcParse/main.cpp (perhaps around line 444 and 629) plus putting the wing identification in L0G-101086.psm1's Convert-Boss-To-Wing function. But I just hacked together a temporary solution that worked in my fork and thought I'd post it in case anyone wants to take the support further.

**Comment by jacob-keller** [comment]
Yea, this would just be a matter of adding the necessary info, as well as modifying your arcdps config to enable these IDs.

I believe if you've enabled the IDs, it will function without names, but we need to add the names to the list for that to function properly.

**Comment by pickettd** [comment]
Yeah, the behavior I was seeing previously was that since the ids were not included for the non-CM fights, the scripts would classify everything as raids but also not know the wings. And the fights would be titled "Unknown encounter "+ bossId. Thanks for being so responsive and updating the code!!

**Comment by jacob-keller** [comment]
Is it possible to confirm whether the pull request i made is working for you? :)

**Comment by jacob-keller** [comment]
https://github.com/jacob-keller/L0G-101086/pull/30

**Comment by pickettd** [comment]
Sorry for the delay in response - my static group has been on break the last two weeks :)
I'll plan to test the PR version tomorrow and let you know. Thanks again!

**Comment by jacob-keller** [comment]
If you need an updated copy of the simpleArcParse binary let me know

**Comment by pickettd** [comment]
Ah yes - an updated binary of simpleArcParse for the PR would be great. I haven't tried building the simpleArcParse solution

**Comment by pickettd** [comment]
Or is there a set of instructions for building simpleArcParse (I'm assuming with Visual Studio 2017 since that is what the solution file is for)? I tried opening the included solution file but it gave me an error that there wasn't a project file. So I tried opening the folder and having it create a project but then there were other errors that came up. Sorry most of my development is in Linux these days so I'm not super familiar with standard practices in Visual Studio. Thanks again!

**Comment by jacob-keller** [comment]
I normally use code blocks. I apologize for not having made the build for
you yet :(

I think it doesn't build in visual studio due to some issues with depending
on gcc compiler builtins...

**Comment by pickettd** [comment]
I installed CodeBlocks with the MinGW compiler suite. I created a new project for simpleArcParse and added the main.cpp file, json.hpp, and the single_include json header (making sure to get the file from the git submodule). It also looks like I needed to enable the c++1 compiler setting. And now it looks the build worked for simpleArcParse.exe (just writing notes in case anyone else is looking for instructions on building it or for my future reference heh).

I'll try running the PR code with my static fractal group tomorrow. Thanks!

**Comment by pickettd** [comment]
I tried the PR version with my latest logs and the changes look like they worked.

For a separate issue though, I noticed that simpleArcDps is not detecting fight success the same way that EliteInsights is detecting success though. Which is why in my branch I used the success field that was returned in the upload result. I'll attach screenshots of what my discord posts look like using my master branch versus using the PR build. Thanks again for all your work!

PR build report: 
![image](https://user-images.githubusercontent.com/410782/71140919-b99c1580-21b6-11ea-84d0-54e6a541d183.png)

pickettd master branch report:
![image](https://user-images.githubusercontent.com/410782/71140950-d1739980-21b6-11ea-8a5f-99afe5e869a3.png)
